### DISCUSSION TITLE
brew update jmeter --with-plugins not working

### DISCUSSION THREAD

**Discussion by rvillane** [Homebrew/discussions#12] [Everyday usage] [answered]
tried to install jmeter 5.3 formula with plugins and got the following error:

`Error: invalid option: --with-plugins`

I'm using:

```
crmar@crmac10-lp 5.3 $ brew -v
Homebrew 2.5.2
Homebrew/homebrew-core (git revision 30caa; last commit 2020-09-24)
```

Any ideas ?

**Comment by SMillerDev** [comment, marked as answer]
Options were removed in 2018, this is intentional

**Reply by rvillane** [reply]
thanks

**Comment by narmadha2851991** [comment]
How to Install JMeter 

`brew install jmeter`

Download the plugin from https://jmeter-plugins.org/wiki/PluginInstall/

Copy the plugin to lib/ext folder

`cp tag-jmeter-extn-1.1.jar /opt/homebrew/Cellar/jmeter/5.6.3/libexec/lib/ext`

Run command Jmeter to restart Jmeter with plugin
